jekyll/_cci2/large-language-model-evaluation.adoc

```diff
@@ -0,0 +1,97 @@
+---
+contentTags:
+  platform:
+    - Cloud
+    - Server v4.x
+---
+= Large Language Model evaluation
+:description: Docs page on setting up LLM evaluation in your CI/CD pipeline
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:experimental:
+
+This page describes common methods for running Large Language Model (LLM) evaluations on CircleCI.
+
+== Evaluations overview
+
+Evaluations, also known as evals, are a methodology for assessing the quality of AI software.
+
+Evaluations provide insights into the performance and efficiency of applications based on LLMs, and allow teams to quantify how well their AI implementation works, measure improvements, and catch regressions.
+
+The term _evaluation_ initially referred to a way to rate and compare AI models, but it has expanded to include application-level testing, including Retrieval Augmented Generation (RAG), function calling, and agent-based applications.
+
+The evaluation process involves using a dataset of inputs to an LLM or application code, and a method to determine if the returned response matches the expected response.  There are numerous evaluation methodologies, including:
+
+* Intrinsic and extrinsic evaluations
+* Human evaluations
+* Bias and fairness checks
+* Readability evaluations
+
+Evaluations can cover many aspects of a model performance, including:
+
+* Ability to comprehend specific jargon
+* Make accurate predictions
+* Avoid hallucinations and generate relevant content
+* Respond in a fair and unbiased way, and within a specific style
+* Avoid certain
```

Comment:
@BeFunes there is something missing here from the original after "avoid certain ...."

Reply:
Good catch! Avoid certain expressions